test/test_util.ts

```diff
@@ -0,0 +1,21 @@
+import * as fs from "fs"
+import { ExampleParser } from "../src/index"
+
+export function parseFile (txtPath: string) {
+    // Load a text to test from the FS
```

Comment:
About `txtPath`: it is a good practice to pass into functions like this **paths, relative to some fixed directory**, and then resolve it with `path.resolve(__dirname, ...paths)`.

```ts
function parseFile(txtPath: string) {
  const txtPathResolved = path.resolve(__dirname, './samples',txtPath)
  // ...
}

parseFile('code_section.py')
```